### PR TITLE
staircase openings

### LINKED ISSUE #151: [mail report] Window in staircase crashes CFAST
Bug report from user.

CFAST input file seems to be broken when there are windows open in staircase. To be investigated.

### PR DESCRIPTION
**Pull request by kowalskiw**
Resolves #151 and covers all possible bugs of this kind in stairwells.

### CODE REVIEW

File: montecarlo/cfast_mcarlo.py [hunk 1]
```diff
@@ -231,7 +237,7 @@ def _section_doors_and_holes(self):# {{{
             vent_to_name = v['vent_to_name'].split(".")[0]
```
Comment (AleksanderKamienski) [inline montecarlo/cfast_mcarlo.py:237]: splitting "." is already done here

File: montecarlo/cfast_mcarlo.py [hunk 2]
```diff
@@ -254,7 +260,7 @@ def _section_vvent(self):# {{{
...
```
Comment (AleksanderKamienski) [inline montecarlo/cfast_mcarlo.py:260]: splitting . is already done here

**Review by AleksanderKamienski — CHANGES_REQUESTED** [review]

File: montecarlo/cfast_mcarlo.py (hunk 1 above)
Comment (kowalskiw) [reply montecarlo/cfast_mcarlo.py:237]: Ok, I will unify processing vent_to_name and vent_from_name across this class. Thanks!

Comment (kowalskiw) [reply montecarlo/cfast_mcarlo.py]: fixed

File: montecarlo/cfast_mcarlo.py (hunk 2 above)
Comment (kowalskiw) [reply montecarlo/cfast_mcarlo.py]: fixed in https://github.com/aamks/aamks/pull/153/commits/a17293b11a592ef6bfbf60972eec5c4aa479cae6

**Review by AleksanderKamienski — APPROVED** [review]

**Pull request merged**